core/service/Cargo.toml

```diff
@@ -29,7 +29,7 @@ consensus_common = { package = "substrate-consensus-common", path = "../../core/
 network = { package = "substrate-network", path = "../../core/network" }
 chain-spec = { package = "substrate-chain-spec", path = "../chain-spec" }
 client = { package = "substrate-client", path = "../../core/client" }
-client_db = { package = "substrate-client-db", path = "../../core/client/db", features = ["kvdb-rocksdb"] }
+client_db = { package = "substrate-client-db", path = "../../core/client/db" }
```

Comment:
Hmm, this might break `node-template` and Polkadot.

Reply:
I added `rocksdb` as an enabled-by-default feature on `substrate-service`. When you import the service, you have to explicitly say `default-features = false` if you don't want rocksdb. This should be fairly fail-safe.

For context, what happens in the worst case scenario is an error at initialization, so it's not that big of a deal. What I'm mostly trying to avoid is the surprise factor.